src/DSharpPlus.Examples.Commands.Basics/Program.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Reflection;
+using System.Threading.Tasks;
+using DSharpPlus.Commands;
+using DSharpPlus.Commands.Processors.SlashCommands;
+using DSharpPlus.Commands.Processors.TextCommands;
+using DSharpPlus.Commands.Processors.TextCommands.Parsing;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace DSharpPlus.Examples.Commands.Basics
+{
+    public static class Program
+    {
+        public static async Task Main(string[] args)
+        {
+            ConfigurationBuilder configurationBuilder = new();
+            configurationBuilder.AddEnvironmentVariables("");
+            configurationBuilder.AddCommandLine(args);
+
+            IConfiguration configuration = configurationBuilder.Build();
+            ServiceCollection serviceCollection = new();
+            serviceCollection.AddSingleton(configuration);
+            Assembly currentAssembly = typeof(Program).Assembly;
+
+            serviceCollection.AddSingleton(async serviceProvider =>
+            {
+                DiscordClient client = new(new DiscordConfiguration()
+                {
+                    Token = configuration.GetValue<string>("discord:token") ??
```

Comment:
consider renaming "discord" in the eaxmples to "example-bot", to make clear what this identifier is

Reply:
It's grabbing the discord token from the discord property:
```json
{
   "discord":
   {
        "token": "invalid"
    }
}
```